### PR TITLE
Fixes #4874 - Add Note About Case Sensitivity

### LINKED ISSUE #4874: Add Note to Example 4 About Case Sensitivity
As PowerShell is usually case-insensitive, I think it’s important to mention in the ‘Example 4: Rename multiple files’ section that when using the ‘replace’ method in this way, that it is case-sensitive.

In the below example, **$_.Name.Replace('Aaa', 'XYZ')** will only replace names that match the exact case (Aaa); it won’t replace aaa or AAA.

`Get-ChildItem *.txt | Rename-Item -NewName { $_.Name.Replace('Aaa', 'XYZ') } -WhatIf`

However, when using the ‘-Replace’ operator in the below way, it is case-insensitive.

`Get-ChildItem *.txt | Rename-Item -NewName { ($_.Name -replace 'Aaa', 'XYZ') } -WhatIf`

And therefore, you would use the ‘-Creplace’ operator if you wanted to match the exact case.

`Get-ChildItem *.txt | Rename-Item -NewName { ($_.Name -creplace 'Aaa', 'XYZ') } -WhatIf`

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: af35d8f3-f95c-f382-b94a-4b89f8b6f8d1
* Version Independent ID: 0e1d944b-6c55-89ed-8e9a-5e5532dba818
* Content: [Rename-Item (Microsoft.PowerShell.Management)](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/rename-item?view=powershell-6#feedback)
* Content Source: [reference/6/Microsoft.PowerShell.Management/Rename-Item.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Management/Rename-Item.md)
* Product: **powershell**
* Technology: **powershell-cmdlets**
* GitHub Login: @daxian-dbw
* Microsoft Alias: **dongbow**

### PR DESCRIPTION
**Pull request by sdwheeler**
Fixes [AB#1610025](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1610025)
Fixes #4874 - Add Note to Example 4 About Case Sensitivity

Version(s) of document impacted
------------------------------
- [x] Impacts 7 document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

### CODE REVIEW

File: reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md [hunk 1]
```diff
@@ -138,18 +153,19 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: False
+Default value: None
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md]: Change all switch parameters default to **False**.

File: reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md [hunk 2]
```diff
@@ -48,67 +48,83 @@ Rename-Item -Path "c:\logfiles\daily_file.txt" -NewName "monday_file.txt"
...
 
-This example shows that you cannot use `Rename-Item` to both rename and move an item.
-Specifically, you cannot supply a path for the value of the **NewName** parameter, unless the path is identical to the path specified in the **Path** parameter.
-Otherwise, only a new name is permitted.
+You can't use `Rename-Item` to both rename and move an item. Specifically, you can't supply a path
+for the value of the **NewName** parameter, unless the path is identical to the path specified in
+the **Path** parameter. Otherwise, only a new name is permitted.
 
-This command attempts to rename the "project.txt" file in the current directory to "old-project.txt" in the "D:\Archive" directory. The result is the error shown in the output.
+This example attempts to rename the "project.txt" file in the current directory to "old-project.txt"
```
Comment (davidsmatlak) [inline reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md]: In paragraph text, replace quotes with backticks. Same in Example 3 and 4.

File: reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md]: I thought I had bulk fixed all of these. 
Fixed

File: reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md (hunk 2 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Rename-Item.md]: Fixed

**Pull request merged**